di-ipv-ticf-stub/lambdas/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "ticf-stub",
+  "description": "TICF CRI stub",
+  "version": "1.0.0",
+  "private": true,
+  "author": "IPVCore",
+  "license": "GDS",
+  "dependencies": {
+    "@aws-lambda-powertools/parameters": "^1.16.0",
+    "@aws-sdk/client-ssm": "^3.451.0",
+    "@types/aws-lambda": "^8.10.109",
+    "aws-sdk": "^2.1497.0",
+    "jose": "4.11.1",
+    "stub-oauth-client": "github:govuk-one-login/stub-oauth-client#main",
+    "ts-node": "^10.9.1"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.23.3",
+    "@babel/preset-env": "^7.23.3",
+    "@babel/preset-typescript": "^7.23.3",
+    "@types/axios": "^0.14.0",
+    "@types/jest": "^29.5.10",
+    "@typescript-eslint/eslint-plugin": "^6.12.0",
+    "@typescript-eslint/parser": "^6.12.0",
+    "axios": "^1.6.2",
+    "babel-jest": "^29.7.0",
+    "eslint": "^8.54.0",
+    "jest": "^29.7.0",
+    "typescript": "^5.3.2"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "test": "jest --config=jest.config.test.js",
+    "integ-test": "jest --config=jest.confg.integ.test.js",
```

Comment:
Are you planning to write integration tests for this?

Reply:
will remove - earlier added was trying to create integration test agnst local run.